### PR TITLE
[15.0][FIX] sale_order_type: Empty analytic account when sale order is duplicated

### PR DESCRIPTION
**Pull request by sergio-teruel**
cc @Tecnativa TT50860

ping  @carlosdauden @carlos-lopez-tecnativa

### CODE REVIEW

File: sale_order_type/models/sale.py [hunk 1]
```diff
@@ -147,6 +147,13 @@ def _prepare_invoice(self):
             res["sale_type_id"] = self.type_id.id
         return res
 
+    def copy_data(self, default=None):
+        vals_list = super(SaleOrder, self).copy_data(default)
```
Comment (carlos-lopez-tecnativa) [inline sale_order_type/models/sale.py]:
```suggestion
        vals_list = super().copy_data(default)
```

**Review by carlos-lopez-tecnativa — APPROVED** [review]
LGTM Only a cosmetic change if possible

**Comment by sergio-teruel** [comment]
Changes done!!

**Review by carlosdauden — APPROVED** [review]

**Pull request merged**